### DISCUSSION TITLE
docker hub repository "spaceone/launchpad" seems empty.

### DISCUSSION THREAD

**Discussion by parkyongsuk** [cloudforet-io/community#61] [Q&A]
https://github.com/cloudforet-io/launchpad/blob/master/dockerfile

the base image is located in "FROM spaceone/launchpad:base"

But the docker repository seems empty. (https://hub.docker.com/r/spaceone/launchpad)

Where can I find the launchpad base docker image?

**Comment by GeunjeLEE** [comment]
Hi! @parkyongsuk! Thank you for letting us know.

We investigated the problem and found out that the base image was deleted by our eco-system.

Now, We have re-registered the base image and you can find & use it.

---

**Notice**
We are planning to implement launchpad so that it can be installed more easily.

**Reply by parkyongsuk** [reply]
Thank you,  Installation is now in progress. However, at the EKS installation stage, an error occurs saying that there is no aws cli.
So, I added the following RUN command to the dockerfile and created the base image again.

FROM spaceone/launchpad:base

RUN apt-get update
RUN apt-get install -y unzip
RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
RUN unzip awscliv2.zip
RUN ./aws/install

WORKDIR /spaceone

ENTRYPOINT ["./gobin"]
